docs/frontend/testing/frontend.md

```diff
@@ -127,8 +86,29 @@ Historically we have leaned on Browser tests to cover testing our app thoroughly
 
 ### Storybook Testing
 
+:::info More Happo documentation
+There is [more information on Happo][docs-internal-happo] and how it's used in
+our Continuous Integration and Continuous Delivery tools section.
+
+[docs-internal-happo]: ../../tools/cicd/happo.md "More Happo documentation"
+:::
+
 * We use [Happo](https://happo.io/) for visually testing Storybook components.
 * Happo will run automatically as a required check on open PRs. If Happo catches any visual diffs with existing components, it will fail and require a review. Anyone at Truss _can_ view the report on Happo and approve or reject changes, but this action should be completed by the designer or PM reviewing the PR for acceptance. When someone accepts or rejects a report, their name and the result will be reported back to the Github PR status.
+
+:::caution Storybook Addon Knobs
+The current version of Storybook used by Happo does not support [the library
+`@storybook/addon-knobs`][npm-storybook-addon-knobs] as expected. While it may
+work locally, Happo will not properly render the contents of components that are
+configured using these **Knobs**.
+
+> 👀 Because of this, make sure that you are passing actual JavaScript Object
+> de-structuring when setting your objects for Storybook components rather than
+> the `object()` function provided by the `@storybook/addon-knobs` library
```

Comment:
I'm looking for extra scrutiny here because I want to make sure that this is as clear as the commit that inspired this callout. https://github.com/transcom/mymove/pull/7840/commits/4a9fd6bab5a1dcb3143ac7f7c19ecef6e4ca798a

Reply:
I'd rephrase this to: 
"pass in actual JavaScript Objects in your component props rather than using the `object()` function"

Reply:
<img width="903" alt="Captura de Pantalla 2021-12-08 a la(s) 11 41 35 a m" src="https://user-images.githubusercontent.com/706004/145247897-3f74c208-d020-4da5-95cc-b41191dbc309.png">

I've updated the admonition a bit to include a code example so it's easier to understand.